tests/PackageServiceProvider.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Vinkla\Tests\Translator;
+
+use Illuminate\Support\ServiceProvider;
+
+class PackageServiceProvider extends ServiceProvider
```

Comment:
Please rename this to `DatabaseServiceProvider` and add your author info to the class. Please also add the license in the top of the file.